HelpSource/Classes/ExampleFiles.schelp

```diff
@@ -59,3 +59,6 @@ TABLE::
 ::
 
 returns:: The full path of TELETYPE::a11wlk01.wav:: as a LINK::Classes/String::
+
+METHOD:: folder
+returns:: The full path of code::examples:: folder as a LINK::Classes/String::
```

Comment:
IMO it would be good to state that this dir is about `sclang`/`scd` examples as all other mentioned files here are sound files.